src/test/java/com/hubspot/jinjava/lib/tag/IncludeTagTest.java

```diff
@@ -188,4 +188,16 @@ public void itSetsErrorLineNumbersCorrectlyTwoLevelsDeep() throws IOException {
     assertThat(result.getErrors().get(0).getSourceTemplate().get())
       .isEqualTo("tags/includetag/errors/error.html");
   }
+
+  @Test
+  public void itRunsWithoutTagCycleException() throws Exception {
```

Comment:
maybe explain this better in the name. Something about extended files not triggering tag cycle exceptions?

Reply:
Sure, renamed to "itAvoidsTagCycleExceptionInsideExtendedFiles"